src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -39,6 +39,7 @@
  * Maintains a set of check suppressions from {@code @SuppressWarnings} annotations.
  * It allows to prevent Checkstyle from reporting violations from parts of code that were
  * annotated with {@code @SuppressWarnings} and using name of the check to be excluded.
+ * It is possible to suppress all the checkstyle warnings with the argument <code>"all"</code>.
```

Comment:
`<code>"all"</code>` => `{@code "all"}`
HTML tags should be used only in xdocs.

Reply:
done